### PR TITLE
Fix workspace patch

### PR DESCRIPTION
**Pull request by taiki45**
Follow upstream istio-proxy changes: upstream's 3781be0e624470a4ee613c4ffc9816976f5dd701

Failed build id: b7f9a5e5-7a31-429c-a743-efd104b77695

### CODE REVIEW

File: envoy_pkg/workspace_patches/istio-proxy/0000-WORKSPACE.patch [hunk 1]
```diff
@@ -13,20 +13,20 @@
 # limitations under the License.
 
 diff --git a/WORKSPACE b/WORKSPACE
-index a9c0baa..1b956aa 100644
+index 22b90ef..4b8325e 100644
 --- a/WORKSPACE
 +++ b/WORKSPACE
-@@ -14,11 +14,17 @@
+@@ -14,12 +14,16 @@
```
Comment (lizan) [inline envoy_pkg/workspace_patches/istio-proxy/0000-WORKSPACE.patch]: can you make this as a new patch? (i.e. 0001-WORKSPACE.patch), the workspace script is trying patches in descending order, so the old workspace will still get patched correctly.

Comment (taiki45) [reply envoy_pkg/workspace_patches/istio-proxy/0000-WORKSPACE.patch]: Sounds good. I’ll change this.

Comment (taiki45) [reply envoy_pkg/workspace_patches/istio-proxy/0000-WORKSPACE.patch]: 98cdd5e

**Comment by taiki45** [comment]
With variant=istio-proxy, the patche files will be applied, but with variant=envoy, the `envoy_pkg/workspace_patches/envoy/*.patch` won't be used. Instead, this path https://github.com/tetratelabs/getenvoy-package/blob/master/envoy_pkg/python/getenvoy/workspace.py#L131-L148 will be used. Do we need `envoy/*.patch`? or we can remove them?

**Comment by taiki45** [comment]
Ah, missing a sign in the revert commit. Will rebase the tree...

**Comment by lizan** [comment]
> With variant=istio-proxy, the patche files will be applied, but with variant=envoy, the envoy_pkg/workspace_patches/envoy/*.patch won't be used.

No, if you build old enough envoy the patches will be used.

**Comment by taiki45** [comment]
I got what the original code does. Fixed.

**Comment by taiki45** [comment]
The CI error happens also in current master... That was caused by recent homebrew code change (they started using ruby 2.6 and its syntax). I'll fix this too.

**Pull request merged**